packages/block-library/src/block/edit.js

```diff
@@ -46,9 +49,22 @@ export default function ReusableBlockEdit( { attributes: { ref }, clientId } ) {
 				'wp_block',
 				ref,
 			] );
+
+			const {
+				getSelectedBlockClientId,
+				getBlockName,
+				getBlockParents,
+			} = select( blockEditorStore );
+
+			const currentBlockId = getSelectedBlockClientId();
+			const parents = getBlockParents( currentBlockId );
+			const _firstParentClientId = parents[ parents.length - 1 ];
```

Comment:
I guess if the reusable block is used inside a columns block for example this logic would not work?

Reply:
Yes, it does work when reusable blocks are inside columns.